src/cli.ts

```diff
@@ -6,18 +6,12 @@ import { parseOpenRPCDocument } from "@open-rpc/schema-utils-js";
 
 program
   .usage("[options] <clientName>")
```

Comment:
usage still says `<clientName>` but below you removed it and just used `process.cwd()`